### PR TITLE
[Web][API] Deprecate Gesture and Activity API

### PR DESCRIPTION
**Pull request by rwalczyna**
[ACR] https://code.sec.samsung.net/jira/browse/TWDAPI-268

Signed-off-by: Rafal Walczyna <r.walczyna@samsung.com>

### CODE REVIEW

**Review by pkosko — APPROVED** [review]

**Comment by arkadiuszpietraszek** [comment]
LGTM

File: docs/application/web/guides/sensors/ham.md [hunk 1]
```diff
@@ -423,12 +365,10 @@ The following table introduces the available monitor types and lists the monitor
...
 | Sleep monitor                        | `http://tizen.org/feature/sensor.sleep_monitor` | When the sleep sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
 | Sleep detector                        | `http://tizen.org/feature/sensor.sleep_monitor` | When the sleep sensor is started, a change callback is invoked when data changes. Use the `getHumanActivityData()` method to get the current data. |
-| Activity recognition                 | `http://tizen.org/feature/sensor.activity_recognition` | To recognize an activity, start listening for it using the `addActivityRecognitionListener()` method.  The following activity types can be recognized:<br> - `STATIONARY`<br> - `WALKING`<br> - `RUNNING`<br> - `IN_VEHICLE` |
```
Comment (mijin85cho) [inline docs/application/web/guides/sensors/ham.md:431]: Just a question. In web/tutorials/app-filtering.md file, we list up the features in a table.

In that table `http://tizen.org/feature/sensor.wrist_up` is included. Don't we need to delete that?

**Review by mijin85cho — COMMENTED** [review]
Below are my questions with this update. Would you please check?

File: docs/application/web/api/6.0/device_api/wearable/tizen/humanactivitymonitor.html [hunk 2]
```diff
@@ -532,19 +527,19 @@ <h4>Attributes</h4>
...
                                        optional boolean? alwaysOn) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
-    void removeGestureRecognitionListener(long watchId) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
-<span class="nocode deprecated">    long addStressMonitorChangeListener(<a href="#StressMonitorDataRange">StressMonitorDataRange</a>[] ranges, <a href="#StressMonitorCallback">StressMonitorCallback</a> listener) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+</span><span class="nocode deprecated">    void removeGestureRecognitionListener(long watchId) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
+</span><span class="nocode deprecated">    long addStressMonitorChangeListener(<a href="#StressMonitorDataRange">StressMonitorDataRange</a>[] ranges, <a href="#StressMonitorCallback">StressMonitorCallback</a> listener) raises(<a href="tizen.html#WebAPIException">WebAPIException</a>);
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/api/6.0/device_api/wearable/tizen/humanactivitymonitor.html:542]: Stress monitor was deprecated earlier but it was not removed from guide. I will remove it.

**Comment by rwalczyna** [comment]
Thank you for detailed review, I updated the commit.

**Review by mijin85cho — COMMENTED** [review]
Thanks for the update, @rwalczyna.

Please get confirmation for the updated version from @pkosko again.

File: docs/application/web/guides/sensors/sensors.md [hunk 3]
```diff
@@ -6,17 +6,13 @@ The following sensor features apply in **mobile and wearable applications only**
...
 
   You can read and manage data from various sensors on the device, and also receive notifications when the sensor data changes. You can access information from various environmental sensors, such as the light and magnetic sensors, and from user-related sensors, such as the heart rate monitor.
 
-- [Gesture Recognition](./ham-gesture.md)
-
-  You can retrieve information about recognized gestures that the user performs with a device.
-
 ## Related Information
-* Dependencies  
+* Dependencies
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/web/guides/sensors/sensors.md:16]: Reminder. Would you please check this comment?

**Comment by pkosko** [comment]
Looks good to me.

**Comment by mijin85cho** [comment]
> Looks good to me.

Thanks for the comment, I'll consider this comment as @pkosko's approval.

**Review by mijin85cho — APPROVED** [review]
Thanks for the update and checking.

**Pull request merged**